src/content/getting-started/secure-your-app/_index.md

```diff
@@ -1,29 +1,159 @@
 
 ## Requirements
 
-<requirements>
+Before start, you need a running workload cluster. If you don't have one, please first [create a workload cluster]({{< relref "/getting-started/provision-your-first-workload-cluster" >}}).
+
+As a next step, you need to deploy an application, here the `hello-world` application, to the workload cluster. Please follow the tutorial [Installing an application]({{< relref "/getting-started/install-an-application" >}}) for that.
+
+## Step 1: Understand the security baseline
+
+By default the clusters come with a `restricted` set of policies that apply to all the namespaces in the cluster. You can read more about which policies are part of the `restricted` set in [Default policies]({{< relref "/vintage/advanced/security/security-policy-enforcement/#default-policies" >}}).
+
+As brief summary, the `restricted` set of policies includes:
+
+- Containers can only run with `non-root` users
+- Not allowing privileged containers
+- Not allowing host namespaces
+- Allowing only a set of capabilities (indeed, only `NET_BIND_SERVICE` is allowed)
+- `HostPath` volumes aren't allowed
+- Only default `Seccomp` and `AppArmor` profiles are allowed
+- `ReadOnlyRootFilesystem` is enforced
+
+In case your application needs to use any of the features that aren't allowed by the `restricted` policies, you need to create a policy exception.
+
+## Step 2:  Configure the security context
+
+In the `hello-world` application, the `restricted` policies aren't violated. The application is running with a `non-root` user and it's not using any special capability of the system (e.g. access to raw sockets).
+
+In `Kubernetes`, the security context is defined in the `Pod` resource. Even if you don't need a special security configuration, you still need to add a minimal security context to your `Pod` resource. Inside the `Pod` manifest, you will find a general `securityContext` which is applied to all the containers within the resource. Additionally, you can define a `securityContext` for each container.
+
+Inspecting the `hello-world` application, you can see pods are created without general `securityContext` but with a `securityContext` for the container.
+
+```yaml
+spec:
+  containers:
+  - name: hello-world
+    # [...]
+    securityContext:
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop:
+        - ALL
+      runAsNonRoot: true
+      runAsUser: 1000
+      seccompProfile:
+        type: RuntimeDefault
+```
+
+Those are necessary to be compliant and pass the validation of the policy engine.
+
+## Step 3:  Create a policy exception
+
+Let’s assume the `hello-world` application needs to run as a `root` user because the container image is designed this way. You don't have more time and your CTO is pushing you to deploy the application.
+
+First, you need to modify the deployment to configure the `securityContext` to allow the container to run as a `root` user.
```

Comment:
Do we want to show this in the hello-world-app way (edit Helm values in config map), edit the `Deployment`, or leave it unexplained how to edit?